src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -237,7 +237,7 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
     }
 
     private boolean checkLineValidity(String line) {
-        return (line.length() >= 5) ? line.contains("-") && (line.charAt(4) == '-') : false;
+        return (line.length() >= 5) ? (line.charAt(4) == '-') : false;
```

Comment:
`return ((line.length() >= 5) && (line.charAt(4) == '-'))`  - why not as simple as that? :wink: